connector/dir.go

```diff
@@ -1,6 +1,8 @@
 package connector
 
 import (
+	"path/filepath"
```

Comment:
This project uses dependency injection, so we would have to create a filepath wrapper for testing.